lhotse/audio.py

```diff
@@ -1323,6 +1323,9 @@ def add_to_mix(
         if audio.size == 0:
             return  # do nothing for empty arrays
 
+        if self.reference_energy <= 0.0:
```

Comment:
hmmm in that case we have audio data with all zeroes, and want to mix something in -- the result should be mixed in audio rather than all zeros. I would remove this `if` here